src/Scene.js

```diff
@@ -15,6 +15,61 @@ type SceneProps = {
 };
 
 class Scene extends React.Component<SceneProps, {}> {
+
+    drawGrid(numRow: number, numColumn: number, width: number, minX: number, minY: number) {
+        const grid = [];
+        let xOffset = minX;
+        let yOffset = minY;
+        for (let i=1;i < numRow + 1;i++) {
+            yOffset = yOffset + width;
+            if (i < numRow) {
+            grid.push(<line
+                className='Scene__grid-line'
+                key={`grid-cell-row-${i}`}
+                x1={`${xOffset}`}
+                y1={`${yOffset}`}
+                x2={`${-xOffset}`}
+                y2={`${yOffset}`} />);
+            }
+            grid.push(
+                <text
+                    className='Scene__grid-label'
+                    textAnchor='end'
+                    key={`grid-cell-label-${i}`}
+                    dominantBaseline='middle'
+                    x={`${xOffset * 1.05}`}
```

Comment:
This calculation also assumes that the origin is in the middle of the Scene. I would suggest instead calculating an offset relative to the width of the Scene (which is in effect what this calculation is doing, but do it more explicitly):

    minX - sceneWidth * 0.025